extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -754,7 +754,7 @@ protected function handleSkinSettings() {
 		if ( $skin instanceof SkinMonoBook ) {
 		    $this->response->addAsset ('extensions/wikia/Search/monobook/monobook.scss' );
 		}
-		if ( $skin instanceof SkinOasis || $skin instanceof SkinVenus ) {
+		if ( $skin instanceof SkinOasis ) {
```

Comment:
I'm not asking this so you change the line above, but I'm curious why it's used `instanceof` and not `F::app()->checkSkin()`? :confused: